src/style.css

```diff
@@ -2,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
```

Comment:
Be consistent with your margins. Add only top or bottom margins, but not both, to avoid potential margin collapse. Since you're setting `margin-top`, ensure that you're not setting `margin-bottom` on any elements that would be adjacent vertically.